### ISSUE TITLE
Not display image latin char on name

### ISSUE THREAD

**Issue by hildogjr** [microsoft/vscode-livepreview#186]
Issue Type: <b>Bug</b>

The LivePreview plugin doesn't display images insert in a HTML file if the image have special/latin characters on the name.

To reproduce:
1. Create a HTML file;
2. Insert a image with "á" in the name;
3. Try to use the LivePreview in this file.

For my test I used the page https://hildogjr.gitlab.io/unicamp-inventory/ with the "Campus Sustentável.png" file image in.

Extension version: 0.2.11
VS Code version: Code 1.62.3 (ccbaa2d27e38e5afa3e5c21c1c7bef4657064247, 2021-11-17T08:00:36.721Z)
OS version: Linux x64 5.11.0-41-generic
Restricted Mode: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-7500U CPU @ 2.70GHz (4 x 3163)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>opengl: enabled_on<br>rasterization: disabled_software<br>skia_renderer: enabled_on<br>video_decode: disabled_software<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|2, 2, 2|
|Memory (System)|15.53GB (3.57GB free)|
|Process Argv|--unity-launch --crash-reporter-id b1cf6861-45ee-48e6-a61c-8956377084d3|
|Screen Reader|no|
|VM|0%|
|DESKTOP_SESSION|ubuntu|
|XDG_CURRENT_DESKTOP|Unity|
|XDG_SESSION_DESKTOP|ubuntu|
|XDG_SESSION_TYPE|x11|
</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vsreu685:30147344
python383cf:30185419
vspor879:30202332
vspor708:30202333
vspor363:30204092
vswsl492cf:30256860
pythontb:30283811
pythonptprofiler:30281270
vshan820:30294714
vstes263:30335439
vscorecescf:30384386
pythondataviewer:30285071
vscod805cf:30301675
pythonvspyt200:30340761
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
pythonrunftest32:30373476
pythonf5test824:30373475
javagetstartedt:30391933
pythonvspyt187:30373474
vsaa593:30376534
pythonvs932cf:30405812
vscexrecpromptc:30404946
vscop804cf:30404767
vscop341:30404997
vsrem710:30405998

```

</details>

**Comment by chingkingm** [comment]
I also got this error but it was in the Chinese environment. When the path of the image contains Chinese, the image cannot be loaded.
For my test, only img2 can be displayed correctly for the following 3 pictures
```html
<img src="../image/avatar/黯蔷薇_bg.png" alt="img0">
<img src="../image/立绘/anqiangwei.png" alt="img1">
<img src="../image/avatar/anqiangwei.png" alt="img2">
```

**Comment by hildogjr** [comment]
I guess it's related with the path string encoding, just working when it's pure ASCII (not allow expanded or UTF-8).

**Comment by cuixiping** [comment]
The fix is here:  #166

**Comment by andreamah** [comment]
Duplicate of https://github.com/microsoft/vscode-livepreview/issues/131
